tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -1350,14 +1365,14 @@ def test_tag_with_conflicting_function_alias_with_other_single_grouping(self):
             assert response.status_code == 200
             assert response.data["9001"]["data"][0][1] == [{"count": 7}]
             assert response.data["abc"]["data"][0][1] == [{"count": 3}]
-            assert response.data["Other"]["data"][0][1] == [{"count": 16}]
+            assert response.data["Other"]["data"][0][1] == [{"count": 15}]
```

Comment:
I couldn't reason with why 16 was here or how it worked.

It looks like top 5 from setup() and the 10 events from this test would lead to 15 events.